Dockerfile

```
@@ -67,8 +67,16 @@ ENTRYPOINT [ "/usr/local/bin/dumb-init", "--" ]
 FROM pre-run AS odbc
 
 USER root
+
+RUN apt update &&  \
```

Generally, to avoid adding redundant layers, all the following lines can be merged into the following:

```suggestion
RUN apt update && \
    apt install -y gnupg && \
    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
    curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
    ACCEPT_EULA=Y apt install -y build-essential unixodbc-dev msodbcsql18
```